macOS/DuckDuckGo/AIChat/AIChatTabOpener.swift

```diff
@@ -54,13 +54,27 @@ struct AIChatTabOpener: AIChatTabOpening {
     @MainActor
     func openAIChatTab(_ value: AddressBarTextField.Value, target: AIChatTabOpenerTarget) {
         let query = addressBarQueryExtractor.queryForValue(value)
-        openAIChatTab(query, target: target)
+
+        // We don't want to auto-submit if the user is opening duck.ai from the SERP
+        // https://app.asana.com/1/137249556945/project/1204167627774280/task/1210024262385459?focus=true
+        let shouldAutoSubmit: Bool
```

Comment:
@Bunn It would be nice to encapsulate this logic somwhere where it can be unit tested.

Reply:
Agree. I'll create an AOR task for this.